opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectHistory.jsx

```diff
@@ -70,7 +70,7 @@ class StixCoreObjectHistory extends Component {
               filters: [
                 { key: 'entity_id', values: [stixCoreObjectId] },
                 {
-                  key: 'event_type',
+                  key: 'event_scope',
```

Comment:
I'm not sure to understand why we should use `event_scope `instead of `event_type`. When I request `logs` in the demo graphql playground, `event_scope` is empty, while `event_type` contains the right values. Example with this report : https://demo.opencti.io/dashboard/analyses/reports/3a5e57e9-fdab-4421-b903-c7edeaf627d1

```
query {
  logs(
    first: 10
    filters: [{key: [entity_id], values: ["3a5e57e9-fdab-4421-b903-c7edeaf627d1"]}]
  ) {
    edges {
      node {
        id
        event_type
        event_scope
        entity_type
        timestamp
      }
    }
  }
}
```

Reply:
When you request the History objects in elastic, you can see that event_type contains a global type (like 'mutation'), and event_scope more precise types (like 'delete', 'update', etc).

Reply:
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/1127c1c0-a08b-47c9-b00e-d9613e0db644)

Reply:
event_type is updated with the value of event_scope, but not at the moment of the query I modified : 
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/83ba0623-4ffd-4a30-988a-b59c20d6f0b3)
(capture of the resolver log.js)

Reply:
but `event_scope` has been introduced recenlty, has it been migrated to have a value for older logs ? From what I can see in the demo, `event_scope` is null for old logs, so the filter will get 0 results for this case

Reply:
ow i'm going to check that